### PR TITLE
Mwalls patch 1

### PR DESCRIPTION
**Pull request by mwalls**

### CODE REVIEW

File: doc/user_guide/getting_started.rst [hunk 1]
```diff
@@ -27,25 +21,35 @@ started as follows:
...
     $ jupyter notebook
 
-Some may find it more convenient to start Jupyter/IPython from the `file manager
-context menu <https://github.com/hyperspy/start_jupyter_cm>`_ or by
-`double-clicking a notebook file <https://github.com/takluyver/nbopen>`_.
+Linux users may find it more convenient to start Jupyter/IPython from the `file manager
+context menu <https://github.com/hyperspy/start_jupyter_cm>`_ . In either OS you can also start by
+`double-clicking a notebook file <https://github.com/takluyver/nbopen>`_ if one already exists.
 
+Importing  HyperSpy into the notebook (or terminal)
```
Comment (francisco-dlp) [inline doc/user_guide/getting_started.rst]: What about "Starting HyperSpy"?

File: doc/user_guide/getting_started.rst [hunk 2]
```diff
@@ -27,25 +21,35 @@ started as follows:
...
+``%matplotlib`` (which is known as a 'python magic') *before executing any plotting command*. So, typically,
+after starting IPython, you can import HyperSpy and set up interactive matplotlib plotting by executing the following
 two lines in the IPython terminal:
 
 .. code-block:: python
 
    In [1]: %matplotlib qt
    In [2]: import hyperspy.api as hs
 
+Note that to execute lines of code in the notebook you must press shift+return. (For details about notebooks and their functionality try the help menu in the notebook).
```
Comment (francisco-dlp) [inline doc/user_guide/getting_started.rst]: If you sourround shift+return with two back-quotes it'll look prettier e.g. ``shift+return``

File: doc/user_guide/getting_started.rst [hunk 3]
```diff
@@ -27,25 +21,35 @@ started as follows:
...
+Importing  HyperSpy into the notebook (or terminal)
+-----------------------------------------------
 Typically you will need to `set up IPython for interactive plotting with
 matplotlib
-<http://ipython.readthedocs.org/en/stable/interactive/plotting.html>`_ using the
-``%matplotlib`` magic *before executing any plotting command*. So, typically,
-after starting IPython, you can import
-HyperSpy and set up interactive matplotlib plotting by executing the following
+<http://ipython.readthedocs.org/en/stable/interactive/plotting.html>`_ using 
+``%matplotlib`` (which is known as a 'python magic') *before executing any plotting command*. So, typically,
```
Comment (jat255) [inline doc/user_guide/getting_started.rst]: These are IPython/Jupyter magic functions, meaning they do not work in a normal python console. Probably better to say `Jupyter magic` or something like that

File: doc/user_guide/getting_started.rst [hunk 4]
```diff
@@ -27,25 +21,35 @@ started as follows:
...
+after starting IPython, you can import HyperSpy and set up interactive matplotlib plotting by executing the following
 two lines in the IPython terminal:
 
 .. code-block:: python
 
    In [1]: %matplotlib qt
    In [2]: import hyperspy.api as hs
 
+Note that to execute lines of code in the notebook you must press shift+return. (For details about notebooks and their functionality try the help menu in the notebook).
+Next, import two necessary modules: numpy and matplotlib.pyplot, as follows:
```
Comment (francisco-dlp) [inline doc/user_guide/getting_started.rst]: Those aren't necessary, but in the rest of the docs we assume that they've been imported.

File: doc/user_guide/getting_started.rst [hunk 5]
```diff
@@ -4,21 +4,15 @@ Getting started
 
 .. _importing_hyperspy-label:
 
-Starting HyperSpy
------------------
+Starting Python in Windows
+----------------------------
+If you used the bundle installation you should be able to use the context menus to get started. Right-click on the folder containing the data you wish to analyse and select "Jupyter notebook here" or "Jupyter qtconsole here". We recommend the former, since notebooks have many advantages over convetional consoles, as will be illustrated in later sections. The examples in some later sections assume Notebook operation. A new tab should appear in your default browser listing the files in the selected folder. To start a python notebook choose "Python 3" in the "New" drop-down menu at the top right of the page. Another new tab will open which is your Notebook.
```
Comment (francisco-dlp) [inline doc/user_guide/getting_started.rst]: > assume Notebook operation

I'm not sure if that's true. I imagine that you want to get rid of the ">>>" to make it less confusing. But those symbols have the nice property that they can be removed from the code examples while the IPython's `In [1]:` can't. Also you could copy and paste the code with the prompts inside the notebook and it'll work. I would rather explain what `>>>` means.

File: doc/user_guide/getting_started.rst (hunk 1 above)
Comment (mwalls) [inline doc/user_guide/getting_started.rst]: Ok I can put that

File: doc/user_guide/getting_started.rst [hunk 6]
```diff
@@ -27,25 +21,35 @@ started as follows:
...
+``%matplotlib`` (which is known as a 'Jupyter magic') *before executing any plotting command*. So, typically,
+after starting IPython, you can import HyperSpy and set up interactive matplotlib plotting by executing the following
 two lines in the IPython terminal:
 
 .. code-block:: python
 
    In [1]: %matplotlib qt
    In [2]: import hyperspy.api as hs
 
+Note that to execute lines of code in the notebook you must press ``shift+return``. (For details about notebooks and their functionality try the help menu in the notebook).
```
Comment (tjof2) [inline doc/user_guide/getting_started.rst]: ``Shift+Return``, i.e. capitalized, could be argued as the convention, on Windows at least? Minor point I know...

**Comment by francisco-dlp** [comment]
It looks ready to merge to me, with the exception of @tjof2 remark. @mwalls, lets us know if you'll address that too.

File: doc/user_guide/getting_started.rst [hunk 7]
```diff
@@ -27,25 +21,35 @@ started as follows:
...
+Next, import two useful modules: numpy and matplotlib.pyplot, as follows: (We use ``>>>`` as a general python propmt)
+
+.. code-block:: python
 
-We also fully support the wx backend. Other backends are supported for plotting
+   >>> import numpy as np
+   >>> import matplotlib.pyplot as plt
+   
+The rest of the documetation will assume you have done this. Now you are ready to load your data (see below).
```
Comment (magnunor) [inline doc/user_guide/getting_started.rst]: Typo: documetation

**Comment by mwalls** [comment]
We can merge it I suppose, but I don't understand the policy for when to put line numbers and when to put arrows....

**Comment by francisco-dlp** [comment]
That inconsistency is again my fault. Because the order is important in that case I used the IPython prompt just there. If you think that it just makes things more confusing feel free to replace them with the standard prompt.

**Comment by mwalls** [comment]
Ok I just suggested a version, which can be merged if we want. Thanks to all for suggestions

**Comment by francisco-dlp** [comment]
Did you make another commit after my previous comment? I can't see it.

**Comment by mwalls** [comment]
I thought I had. I just tried again. It now seems to be doing wierd things to the code lines - unwanted line breaks. I'll try again

**Pull request merged**